components/brave_wallet_ui/common/async/lib.ts

```diff
@@ -48,7 +48,7 @@ export const getERC20Allowance = (
 export const onConnectHardwareWallet = (opts: HardwareWalletConnectOpts): Promise<BraveWallet.HardwareWalletAccount[]> => {
   return new Promise(async (resolve, reject) => {
     const keyring = getHardwareKeyring(opts.hardware)
-    if (keyring instanceof LedgerBridgeKeyring || keyring instanceof TrezorBridgeKeyring) {
+    if (keyring instanceof LedgerKeyring || keyring instanceof TrezorBridgeKeyring) {
```

Comment:
`LedgerKeyring` is an interface, whereas I think `TrezorBridgeKeyring` is an implementation. Can we use the same for both cases?

Reply:
done